### DISCUSSION TITLE
is OTLP/HTTP supported?

### DISCUSSION THREAD

**Discussion by c-cube** [jaegertracing/jaeger#3591] [Q&A] [answered]
Hi, I'm trying to export traces into jaeger using the opentelemetry format, as protobuf, and I'm not sure if the http extension to OLTP is supported?

More precisely:
- running jaeger via the all-included docker daemon:
  ```docker run -d --name jaeger \
  -e COLLECTOR_ZIPKIN_HOST_PORT=:$PORT \
  -p 5775:5775/udp \
  -p 6831:6831/udp \
  -p 6832:6832/udp \
  -p 5778:5778 \
  -p 16686:16686 \
  -p 14250:14250 \
  -p 14268:14268 \
  -p 14269:14269 \
  -p 9411:9411 \
  jaegertracing/all-in-one:1.32
  ```
- connecting on 14269 via http (which is open)
- sending binary encoded protobufs on `/v1/traces`
- getting back a response with code 200 and body… `{"status":"Server available","upSince":"2022-03-16T16:50:40.565423833Z","uptime":"7h14m9.629882443s"}
` instead of the protobuf status I would expect. 

That leads me to believe I'm either doing something wrong, or jaeger currently only supports the gRPC part of opentelemetry. Which is it? :)

**Comment by yurishkuro** [comment, marked as answer]
Jaeger does not support OTLP at this time at all (open issue #3625). You need to configure OTEL collector to export data in Jaeger format (grpc/proto).

**Reply by c-cube** [reply]
Thank you! I've been able to connect to the otelcol locally, will look into hooking it up then :)

**Reply by vitorf7** [reply]
@yurishkuro Sorry this might a totally stupid question as I might have missed it somewhere else. Does jaeger plan to support the OTLP at any point and if so is there anywhere we can keep an eye on it?
Thanks.

**Reply by yurishkuro** [reply]
#3625
